README.md

```diff
@@ -52,6 +52,30 @@ find('192.168.0.10').then(device => {
   }
   */
 })
+
+// Find all devices within 192.168.0.1 to 192.168.0.25 range
+find('192.168.0.1-192.168.0.25').then(devices => {
+    devices /*
+    [
+        { name: '?', ip: '192.168.0.10', mac: '...' },
+        { name: '...', ip: '192.168.0.17', mac: '...' },
+        { name: '...', ip: '192.168.0.21', mac: '...' },
+        { name: '...', ip: '192.168.0.22', mac: '...' }
+    ]
+    */
+})
+
+// Find all devices within /24 subnet range of 192.168.0.x
+find('192.168.0.0/24').then(devices => {
+    devices /*
+    [
+        { name: '?', ip: '192.168.0.10', mac: '...' },
+        { name: '...', ip: '192.168.0.50', mac: '...' },
+        { name: '...', ip: '192.168.0.155', mac: '...' },
+        { name: '...', ip: '192.168.0.211', mac: '...' }
+    ]
+    */
+})
```

Comment:
Please adjust the indentation, so it aligns with the other examples of the README.

```suggestion
// Find all devices within 192.168.0.1 to 192.168.0.25 range
find('192.168.0.1-192.168.0.25').then(devices => {
    devices /*
    [
      { name: '?', ip: '192.168.0.10', mac: '...' },
      { name: '...', ip: '192.168.0.17', mac: '...' },
      { name: '...', ip: '192.168.0.21', mac: '...' },
      { name: '...', ip: '192.168.0.22', mac: '...' }
    ]
    */
})

// Find all devices within /24 subnet range of 192.168.0.x
find('192.168.0.0/24').then(devices => {
    devices /*
    [
      { name: '?', ip: '192.168.0.10', mac: '...' },
      { name: '...', ip: '192.168.0.50', mac: '...' },
      { name: '...', ip: '192.168.0.155', mac: '...' },
      { name: '...', ip: '192.168.0.211', mac: '...' }
    ]
    */
})
```